components/LibraryIndex/LibraryIndex.jsx

```diff
@@ -1,5 +1,24 @@
-function LibraryIndex() {
-	return <></>
+function LibraryIndex(props) {
+	const {libraryItems} = props
+
+	const allLibraryItems = libraryItems.map((libraryItem, index) => (
+		<ul key={index}>
+			<li>Name: {libraryItem.name}</li>
+			<li>Description: {libraryItem.description}</li>
+			<li>Publication Date: {libraryItem.publicatioDate}</li>
+			<li>Author: {libraryItem.author}</li>
+			<li>
+				Reviews: <li>Description: {libraryItem.review.description}</li>
+				<li>Rating: {libraryItem.review.rating}</li>
```

Comment:
I don't think we'll need all this information for the index.

Reply:
agreed - I got rid of this info